### ISSUE TITLE
Generalise genetic programming

### ISSUE THREAD

**Issue by lexvanderstoep** [CvanderStoep/Genetic_Swarm#3]
Generalise the genetic programming approach.

In the future, it could for example be used to solve the Travelling Salesman Problem

**Comment by CvanderStoep** [comment]
Je bedoelt het schrijven van een generieke interface?

**Comment by lexvanderstoep** [comment]
Ja. Het schrijven van een interface en vervolgens de simulatie daar op aanpassen. Ik kan wel een begin maken door de interface te definiëren.

**Comment by lexvanderstoep** [comment]
Has this been implemented yet? Our users have been waiting patiently.

**Comment by CvanderStoep** [comment]
Our users are waiting for the start of the design of the interface by you, see your post of 15/may/2019.
